Studentify/appsettings.Development.json

```diff
@@ -8,5 +8,11 @@
       "Microsoft": "Warning",
       "Microsoft.Hosting.Lifetime": "Information"
     }
-  }
+  },
+  "JWT": {
```

Comment:
Hmm, chcemy to mieć publiczne na branchu? To nie jest jakiś secret przypadkiem?

Reply:
I tak nie ma tam narazie bardzo wrażliwych danych. Jak ustalimy z frontem czy będzie nam to wgl potrzebne przy tokenach to będzie można to ukryć.